### PR TITLE
Update the certificates on the xenial slaves.

### PR DESCRIPTION
**Pull request by elopio**

### CODE REVIEW

File: containers/jenkins-slave-xenial/Dockerfile [hunk 1]
```diff
@@ -36,7 +36,8 @@ RUN apt-get update && apt-get install --no-install-recommends -qy \
   python3-requests-oauthlib python3-requests-toolbelt \
   python3-ssoclient python3-testscenarios python3-testtools \
   python3-xdg python3-yaml \
-  snapcraft && rm -rf /var/lib/apt/lists/*
+  snapcraft && rm -rf /var/lib/apt/lists/* && \
+  update-ca-certificates -f
```
Comment (fgimenez) [inline containers/jenkins-slave-xenial/Dockerfile]: I would put this in a separate `RUN` command, the two actions aren't related and this way the layers can be cached separately.

**Comment by fgimenez** [comment]
Looks good, should we remove this from the master?

**Comment by elopio** [comment]
I moved the statement to a separate run.
Where is this in master?

**Comment by fgimenez** [comment]
> Where is this in master?

It is done in jenkins-master's base image https://github.com/ubuntu-core/jenkins-ubuntu/blob/master/Dockerfile#L5, not sure if we still need this, anyway we can take care of that in another PR, thanks! :+1:

**Comment by elopio** [comment]
@fgimenez right, I remember now. What I don't remember is if it was needed to get some of the jenkins plugins or not. At this point this projects are getting complex, so I think we should start reporting bugs before we make the PRs, or explaining our PRs a lot better.
If you remember the certificate update was only for snapcraft tests, I will go ahead and remove it from master.

**Pull request merged**